From: hyoyeong <164199371@qq.com>
Date: Thu, 3 Sep 2026 03:31:35 +0000
Subject: =?utf-8?b?VHJhbnNsYXRlZCB1c2luZyBXZWJsYXRlICjmnJ3pspzor60gKGtvX0tQ?=
 =?utf-8?b?KSk=?=

Currently translated at 100.0% (91 of 91 strings)

Translation: openkylin-huanghe/libukui-search
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/libukui-search/ko_KP/
---
 translations/libukui-search/libukui-search_ko_KP.ts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/translations/libukui-search/libukui-search_ko_KP.ts b/translations/libukui-search/libukui-search_ko_KP.ts
index e25107e..64e201f 100644
--- a/translations/libukui-search/libukui-search_ko_KP.ts
+++ b/translations/libukui-search/libukui-search_ko_KP.ts
@@ -463,7 +463,7 @@
     <message>
         <location filename="../../libsearch/mailsearch/mail-search-plugin.cpp" line="114"/>
         <source>Mail</source>
-        <translation>메일</translation>
+        <translation>전자우편</translation>
     </message>
     <message>
         <location filename="../../libsearch/mailsearch/mail-search-plugin.cpp" line="251"/>
